Sources/QuickTests/Helpers/QuickSpec+QuickSpec_MethodList.h

```diff
@@ -0,0 +1,7 @@
+#import <Quick/Quick.h>
+
+@interface QuickSpec (QuickSpec_MethodList)
+
++ (NSSet<NSString*> *)allSelectors;
```

Comment:
Add documentation to this method. The documentation should make it clear that:

1. An instance of the class will be instantiated in order to get a method list.
2. This method should only be used for test assertions.